static/Functions.js

```diff
@@ -2,10 +2,9 @@
 // See full license at the bottom of this file.
 
 
-
 // The initialize function is required for all add-ins.
 Office.initialize = function () {
-  jQuery(document).ready(function() {
+  jQuery(document).ready(function () {
```

Comment:
no reason for the space here after the function...
h&e

Reply:
done h&e